src/Integration.Vsix.UnitTests/CFamily/CLangAnalyzerTests.cs

```diff
@@ -99,7 +99,7 @@ public void ExecuteAnalysis_RequestCanBeCreated_AnalysisIsTriggered()
         [TestMethod]
         public void TriggerAnalysisAsync_StreamsIssuesFromSubProcessToConsumer()
         {
-            const string fileName = "c:\\data\\aaa\\bbb\file.txt";
+            const string fileName = "c:\\data\\aaa\\bbb\\file.txt";
```

Comment:
Incorrect escaping in the path - missing \.

It didn't matter before when the Debug.Assert was comparing strings.
However, `PathHelper` uses `Path.GetCanonicalPath`, which validates the characters -> exception.